src/gui/accountmanager.cpp

```diff
@@ -297,10 +297,7 @@ AccountPtr AccountManager::loadAccountHelper(QSettings &settings)
     auto acc = createAccount();
 
     QString overrideUrl = Theme::instance()->overrideServerUrlV2();
-    QString forceAuth = Theme::instance()->forceConfigAuthType();
-    if (!forceAuth.isEmpty() && !overrideUrl.isEmpty()) {
-        // If forceAuth is set, this might also mean the overrideURL has changed.
-        // See enterprise issues #1126
+    if (!overrideUrl.isEmpty()) {
```

Comment:
That does not match the old behaviour.
Remove the setting of overrideUrl here all together.